client/app/bundles/Zoon/components/WorkflowDiagram.jsx

```diff
@@ -27,7 +27,11 @@ const Switch = ({ value, onChange }) => (
 )
 
 const SelectedItem = ({ item, sharedProps }) => (
-  <div className="module-list__list__item">
+  <div
+    className="module-list__list__item"
+    onMouseEnter={() => sharedProps.wantsModuleShown(item.id)}
```

Comment:
I was trying to go against that, but your argument makes total sense, and promotes more reuse. Doing that.